package/client/resource/dui/index.ts

```diff
@@ -0,0 +1,80 @@
+import {
+	GetGameTimer,
+	CreateDui,
+	GetDuiHandle,
+	CreateRuntimeTxd,
+	CreateRuntimeTextureFromDuiHandle,
+	SetDuiUrl,
+	DestroyDui,
+	SendDuiMessage,
+} from '@nativewrappers/client';
+import { cache } from '../cache';
+
+let duis: { [key: string]: Dui } = {};
+let currentId = 0;
+
+interface LibDui {
+	url: string;
+	width: number;
+	height: number;
+	debug?: boolean;
+}
+
+export class Dui {
+	id: string = "";
+	debug: boolean = false;
+	url: string = "";
+	duiObject: number = 0;
+	duiHandle: string = "";
+	runtimeTxd: number = 0;
+	txdObject: number = 0;
+	dictName: string = "";
+	txtName: string = "";
+
+	constructor(data: LibDui) {
+		const time = GetGameTimer();
+		const id = `${cache.resource}_${time}_${currentId}`;
+		currentId++;
+		this.id = id;
+		this.debug = data.debug || false;
+		this.url = data.url;
+		this.dictName = `ox_lib_dui_dict_${id}`;
+		this.txtName = `ox_lib_dui_txt_${id}`;
+		this.duiObject = CreateDui(data.url, data.width, data.height);
+		this.duiHandle = GetDuiHandle(this.duiObject);
+		this.runtimeTxd = CreateRuntimeTxd(this.dictName);
+		this.txdObject = CreateRuntimeTextureFromDuiHandle(this.runtimeTxd, this.txtName, this.duiHandle);
+		duis[id] = this;
+
+		if (this.debug) console.log(`Dui ${this.id} created`);
+	}
+
+	remove = () => {
```

Comment:
Proper method syntax is

```ts
remove() {

}

Reply:
https://github.com/overextended/ox_lib/blob/cba4b85a2958f68a93211676eaeee88fa2d0dcd8/package/client/resource/points/index.ts#L46